### DISCUSSION TITLE
Change only the Clock Flyout.

### DISCUSSION THREAD

**Discussion by FeldNico** [valinet/ExplorerPatcher#2339] [Q&A]
Hi,
I just switched to Windows 11 and is quite a change, but I slowly accustom to the new layouts.
However, I really can't stand that there are no events in the Clock/Calender Flyout anymore.
Is there a way to only use the Clock Flyout from Windows 10 and leave everything else in the Windows 11 style?
Thanks!

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by Amrsatrio** [reply]
This is not an issue. This is a feature request so those info aren't needed. OP didn't say that they're using the Win10 taskbar, and also the fact that they have been getting used to the new UI elements.

**Comment by Amrsatrio** [comment]
There is currently no way to do that at the moment, but perhaps it is possible for us to make EP detect if the user left clicks on the clock within the "omni button" and it'll open the old clock flyout.

**Comment by FeldNico** [comment]
Okay :/
Thank you for answering!

**Comment by 1598247157** [comment]
Hi,
I'd like to revisit a feature request that was previously discussed in https://github.com/valinet/ExplorerPatcher/discussions/2339 ("Change only the Clock Flyout") in https://github.com/valinet/ExplorerPatcher/discussions/2339?utm_source
The request is to allow users to keep the native Windows 11 taskbar, while replacing only the clock/calendar flyout with the Windows 10 version.
The main reason I would like this feature is that the Windows 10 calendar flyout can display calendar events/agenda directly below the calendar, while the Windows 11 flyout no longer provides this functionality.
Ideally, the behavior would be:
Windows 11 taskbar → click the native clock/date → Windows 10-style clock/calendar flyout with agenda/events
I understand that this was not possible when https://github.com/valinet/ExplorerPatcher/discussions/2339 was discussed. However, nearly three years have passed, 和 both Windows 11 and ExplorerPatcher have changed significantly since then.
Would it now be technically possible to implement this as a standalone option, without switching the entire taskbar to the Windows 10 (ExplorerPatcher) taskbar?
If it is still not possible, I'd also be interested to know whether the technical limitations have changed since the original discussion.
Thanks for maintaining ExplorerPatcher!
